### PR TITLE
repoman + other maintenance

### PR DESCRIPTION
**Pull request by gkroon**
I decided to do some maintenance, mostly repoman related. Also removed `profiles/use.local.desc` after moving some local USE flags to the packages' `metadata.xml` (see https://devmanual.gentoo.org/profiles/use.desc/index.html). I also propose to remove `cuda-rarcrypt/cuda-rarcrypt` as it's no longer in active development, cannot be merged any more due to depending on outdated CUDA, and I think hashcat and john are already great substitutes. Not yet removed, though.

I hope it is ok to have spend some time on this? Please reach out to me if anything needs to be changed before it can be merged.

### CODE REVIEW

File: app-admin/vuls/vuls-0.9.0-r1.ebuild [hunk 1]
```diff
@@ -250,10 +253,6 @@ IUSE="policykit systemd"
 SLOT=0
```
Comment (linxon) [inline app-admin/vuls/vuls-0.9.0-r1.ebuild]: add `RESTRICT="mirror"` at this line please

File: app-admin/vuls/vuls-0.9.0-r1.ebuild [hunk 2]
```diff
@@ -262,16 +261,13 @@ RDEPEND="
 	dev-go/goval-dictionary[policykit=]
 	dev-go/gost[policykit=]
 	dev-go/go-exploitdb[policykit=]
-	policykit? ( sys-auth/polkit )
+	policykit? (
+		acct-group/vuls
+		acct-user/vuls
```
Comment (linxon) [inline app-admin/vuls/vuls-0.9.0-r1.ebuild]: maybe it also needs changing for ebuilds of these packages
```
dev-go/go-cve-dictionary
dev-go/gost
dev-go/goval-dictionary
dev-go/go-exploitdb
dev-go/go-exploitdb
www-apps/vulsrepo
```

File: app-admin/vuls/vuls-0.9.0-r1.ebuild (hunk 1 above)
Comment (gkroon) [reply app-admin/vuls/vuls-0.9.0-r1.ebuild]: Added

File: app-admin/vuls/vuls-0.9.0-r1.ebuild (hunk 2 above)
Comment (gkroon) [reply app-admin/vuls/vuls-0.9.0-r1.ebuild]: Thanks, also changed those in 52a80a6 now

**Pull request merged**